mne/io/brainvision/brainvision.py

```diff
@@ -33,9 +33,10 @@ class RawBrainVision(BaseRaw):
     ----------
     vhdr_fname : str
         Path to the EEG header file.
-    montage : str | None | instance of Montage
-        Path or instance of montage containing electrode positions.
-        If None, sensor locations are (0,0,0). See the documentation of
+    montage : str | True | None | instance of Montage
+        Path or instance of montage containing electrode positions. If
+        True, read embedded sensor locations. If None, sensor locations
+        are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
```

Comment:
I agree. in this case I wouldn't consider it a montage in the traditional sense (standard template). it would be more like how we handle MEG data.